crates/diesel_models/src/payment_method.rs

```diff
@@ -46,6 +56,51 @@ pub struct PaymentMethod {
     pub updated_by: Option<String>,
 }
 
+#[cfg(all(feature = "v2", feature = "payment_methods_v2"))]
+#[derive(
+    Clone, Debug, Eq, PartialEq, Identifiable, Queryable, Selectable, Serialize, Deserialize,
+)]
+#[diesel(table_name = payment_methods, primary_key(id), check_for_backend(diesel::pg::Pg))]
+pub struct PaymentMethod {
+    pub customer_id: common_utils::id_type::CustomerId,
```

Comment:
This should the global id and not `common_utils::id_type::CustomerId`, since the customer PR is not merged make it string for now